moon/controller.py

```diff
@@ -157,6 +160,7 @@ def send_request(self, cmd, callback=None):
 
     def set_cam_angle(self, angle):
         self.send_request('set_cam_angle %f\n' % angle)
+        angle = min(math.pi / 4.0, max(-math.pi / 8.0, angle))
```

Comment:
why the check is AFTER the request?

Reply:
you're right of course. Subsequent code changes (specifically reading the tilt from the joint_states instead of assuming it is what the last command said it should be) make it irrelevant either way. It just feels cleaner to send the command with the parameters within the valid range.